lib/workers/repository/dependency-dashboard.ts

```diff
@@ -346,6 +415,20 @@ export async function ensureDependencyDashboard(
           checkText.replace('[ ]', '[x]')
         );
       }
+      if (dependencyDashboardAllPending) {
+        const checkAllPending = ` - [ ] <!-- approve-all-pending-prs -->`;
+        issueBody = issueBody.replace(
+          checkAllPending,
+          checkAllPending.replace('[ ]', '[x]')
+        );
+      }
+      if (dependencyDashboardAllRateLimited) {
```

Comment:
so lets say in this run we replace all [ ] with [x] , that means we marked them rows.
do we also work on them afterwards? or do we need to run again in order to catch the newly marked rows?

Reply:
yes, after marking the select all checkbox when running Renovate we will mark all the relevant branches in the dashboard in the same run.